mtz-ajax-interceptor-behavior.html

```diff
@@ -1,13 +1,136 @@
 <link rel="import" href="../polymer/polymer.html">
 
-<!--
-`mtz.AjaxInterceptorBehavior`
+<script>
+  window.mtz = window.mtz || {};
 
-@demo demo/index.html
--->
+  /**
+   * `mtz.AjaxInterceptorBehavior` handles registering declarative interceptors then listening for all registered events
+   *  and calling the interceptors associated with those events.
+   *
+   * @polymerBehavior mtz.AjaxInterceptorBehavior
+   */
+  mtz.AjaxInterceptorBehavior = {
+    properties: {
+      /* The map of event to interceptor Set to listen against */
+      interceptors: {
+        type: Object,
+        value() {
+          return {};
+        }
+      }
+    },
+    attached() {
+      this.addEventListener('register-interceptor', this.__registerInterceptor);
```

Comment:
Generally, I think we will want to. I'm not sure on this we would want to because these are critical to making requests.